h2o-py/h2o/model/metrics_base.py

```diff
@@ -319,7 +325,8 @@ def aic(self):
         ...                validation_frame = valid)
         >>> pros_glm.aic()
         """
-        return self._metric_json['AIC']
+        if self._algo == 'glm':
+            return self._metric_json['AIC']
```

Comment:
I'd prefer if you'd not care about the algo. So I'd suggest to use `.get("AIC")` instead of checking the `algo`. This way if we expose these metrics e.g. from the metalearner to stacked ensemble or if Wendy adds it to GAM we will not need to modify this part.

```suggestion
        return self._metric_json.get('AIC')
```

Reply:
Removed the algo check per your suggestions.